core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -273,6 +273,8 @@ trait EntityComponent {
       }
     }
 
+    def refIdMap(entityRecords: Seq[EntityRecord]): Map[AttributeEntityReference, Long] = entityRecords.map(e => e.toReference -> e.id).toMap
```

Comment:
I think I'm +1 on Doug's comment here. At minimum the function name has to be verrrrrry descriptive to avoid the train of thought that Doug described. Not sure it's worth it.